hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/AssignmentManager.java

```diff
@@ -231,17 +231,15 @@ public void start() throws IOException, KeeperException {
       RegionState regionState = MetaTableLocator.getMetaRegionState(zkw);
       RegionStateNode regionNode =
         regionStates.getOrCreateRegionStateNode(RegionInfoBuilder.FIRST_META_REGIONINFO);
-      regionNode.lock();
```

Comment:
Donot need lock because the master was started now? No other procedures runed at the same time?

Reply:
No. As you said, no procedures yet and no concurrent call to this method either. The code is written by me but now I think adding the lock here will make people confusing, let's just remove it.

Reply:
> no procedures yet and no concurrent call to this method 
Nit: Add this comment here?

Reply:
Let me add comments. Missed this one.